sycl/include/CL/sycl/intel/fpga_device_selector.hpp

```diff
@@ -32,9 +32,9 @@ class platform_selector : public device_selector {
   }
 };
 
-const std::string EMULATION_PLATFORM_NAME =
+const char *EMULATION_PLATFORM_NAME =
```

Comment:
Is it more C++ now?
```suggestion
static constexpr auto EMULATION_PLATFORM_NAME =
```

Reply:
'auto' should be used only where the type is obvious, and this is not the case, imo.

Reply:
+1 to a comment above. If we really want it be C-plus-plusish - we can use std::array here (I personally don't have such preferences).

Reply:
I don't think it's worth including the std header just for a constant string that is used for one particular case.